src/test/compile-fail/rfc-2008-non-exhaustive/enum_field_attempt_create.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// FIXME issue #44109, PR #45394
+// gate-test-non_exhaustive
+#![feature(non_exhaustive)]
```

Comment:
Unnecessary feature, `feature(non_exhaustive)` is required only in crated defining something `non_exhaustive`, not using it.

Reply:
Same with other tests.

Reply:
Removed the unnecessary feature usages.